### PR TITLE
Add projects data

### PR DESCRIPTION
**Pull request by deyemiobaa**
- Added data of two my projects to data.js file

### CODE REVIEW

File: src/data.js [hunk 1]
```diff
@@ -210,4 +210,46 @@ export default [
...
+    id: 15,
+    title: 'ToDo-List',
+    description: 'This is a website A web app that helps you keep track of your most important tasks. You can add, delete, edits tasks as you wish, and they always stay on your browser.',
+    author: 'Sodiq Aderibigbe',
+    ghUsername: 'deyemiobaa',
+    repo: 'https://github.com/deyemiobaa/ToDo-List',
+    license: 'MIT',
+    screenshot:
+      'https://raw.githubusercontent.com/deyemiobaa/ToDo-List/main/src/images/app-sc.png',
+    publishedOn: '2022-04-21',
```
Comment (ahangarha) [inline src/data.js]: Please put the date of the last commit here. The day you finished your project

File: src/data.js [hunk 2]
```diff
@@ -210,4 +210,46 @@ export default [
...
+    id: 16,
+    title: 'ShowBox Movies',
+    description: 'Search for info about your favorite movies and tv shows on the ShowBox Movie Database',
+    author: 'Sodiq Aderibigbe, Sentayhu berhanu',
+    ghUsername: 'deyemiobaa',
+    repo: 'https://github.com/deyemiobaa/Movie-Database',
+    license: 'MIT',
+    screenshot:
+      'https://raw.githubusercontent.com/deyemiobaa/Movie-Database/dev/src/app-sc.png',
+    publishedOn: '2022-04-21',
```
Comment (ahangarha) [inline src/data.js]: Please put the date of the last commit here. The day you finished your project

File: src/data.js [hunk 3]
```diff
@@ -210,4 +210,46 @@ export default [
...
+    id: 17,
+    title: 'CC-Global-Summit',
+    description: 'The Culture Custodian Global Fashion Summit website is provides information about the event, speakers, and how guests can attend the event',
+    author: 'Sodiq Aderibigbe',
+    ghUsername: 'deyemiobaa',
+    repo: 'https://github.com/deyemiobaa/CC-global-summit',
+    license: 'MIT',
+    screenshot:
+      'https://raw.githubusercontent.com/deyemiobaa/CC-global-summit/050b356a70e6320e11bfab353c1a09ac5826ed49/images/project-snapshot.svg',
+    publishedOn: '2022-04-21',
```
Comment (ahangarha) [inline src/data.js]: Please put the date of the last commit here. The day you finished your project

**Review by ahangarha — CHANGES_REQUESTED** [review]
Thanks for your PR. It follows the correct structure which is perfect. :+1: 

Just please update dates for each project and add the date you finished (submitted) the project.

**Comment by deyemiobaa** [comment]
> Thanks for your PR. It follows the correct structure which is perfect. 👍
> 
> Just please update dates for each project and add the date you finished (submitted) the project.



Thanks, I've updated the dates to reflect the last commits

**Review by ahangarha — APPROVED** [review]
# Approved :tada: 

Thanks for your contribution. Well done. :+1:

**Pull request merged**